addons/website/controllers/main.py

```diff
@@ -535,7 +536,16 @@ def make_scss_custo(self, url, values):
         Returns:
             boolean
         """
-        request.env['web_editor.assets'].make_scss_customization(url, values)
+        # Protect variable names so they cannot be computed as numbers on SCSS
+        # compilation (e.g. var(--700) => var(700)).
+        scss_fixed_values = {}
+        for key, value in values.items():
+            scss_fixed_values[key] = re.sub(
+                r"var\(--([0-9]+)\)",
+                lambda matchobj: "var(--#{" + matchobj.group(1) + "})",
+                value)
+
+        request.env['web_editor.assets'].make_scss_customization(url, scss_fixed_values)
```

Comment:
Maybe worth getting one level lower and do it into the `make_scss_customization` method instead of the controller, as I see that some other exceptions are handled there, what do you think?